### ISSUE TITLE
wild_gt_tcc.pth cannot generate foot contact signal?

### ISSUE THREAD

**Issue by chenjiaxiang** [Shimingyi/MotioNet#35]
I'd like to appreciate for code you released, I want to use the foot contact signal which should be generated in the MotioNet code. But I found in the wild_gt_tcc.pth file, the contact config is false, so I checked code and weight of parameters,  and Branch Q's output feature is 49, I think the feature should be 51 if the code can generate foot contact signal. Here are some pictures i checked.
![截屏2021-03-18 下午1 03 41](https://user-images.githubusercontent.com/19178992/111575965-7b8f0680-87ea-11eb-903c-4d7c9ff76e52.png)
this is the pth file config.
![截屏2021-03-18 下午1 02 23](https://user-images.githubusercontent.com/19178992/111576000-91043080-87ea-11eb-864e-bb247af68bf4.png)
this is the weight shape of branch Q.
![截屏2021-03-18 下午1 02 59](https://user-images.githubusercontent.com/19178992/111576040-a6795a80-87ea-11eb-82ae-e84c47bdf8a7.png)
this is the code in model/model.py.

If I'm right, the pth can not generate foot contact signal, if I'm wrong, pls tell me how to use the pth file to generate foot contact signal. 
And if the pth file can not generate foot contact signal, can you offer a pth file which can generate foot contact signal? I would be appreciated for your sharing.

**Comment by Shimingyi** [comment]
Hi @chenjiaxiang ,

Yes, in the current model the foot signal is turned off because of potential bugs mentioned in #28 for avoiding ambiguity.
If you want to use a version with foot contact prediction, you can follow this command to get an own model file. I will also update another pth file for supporting that.

```
python train.py -n wild -d 1 --kernel_size 5,3,1 --stride 3,1,1 --dilation 1,1,1 --channel 1024 --confidence 1 --translation 1 --contact 1 --loss_term 1101
```

**Comment by chenjiaxiang** [comment]
@Shimingyi OK，I will train a model with foot contact label,  but did you fix the bug you mentioned in  #28 about the foot-contact loss? I worry about the performance of model influenced by the bug.
